dataprofiler/tests/data_readers/test_csv_data.py

```diff
@@ -497,19 +497,6 @@ def test_specifying_data_type(self):
                 input_data_obj.delimiter, input_file["delimiter"], input_file["path"]
             )
 
-    def test_specifying_data_type_when_sampled(self):
```

Comment:
I don't think the functionality covered in this test is affected by reservoir sampling.

Reply:
`input_file["path"], data_type="csv", options={"sample_nrows": 100}` 

are you sure

Reply:
so I see how we triggered reservoir sampling, but I don't see how reservoir sampling affects these assertions since it only affects the data itself not the metadata of the object. Same with the tests below that I removed

Reply:
Got it!